src/commands/pack/tarballs.ts

```diff
@@ -24,6 +24,9 @@ This can be used to create oclif CLIs that use the system node or that come prel
     const {flags} = await this.parse(PackTarballs)
     const targets = flags.targets.split(',')
     const buildConfig = await Tarballs.buildConfig(flags.root, {xz: flags.xz, targets: targets})
+    if (targets.includes('darwin-arm') && !buildConfig.nodeVersion.startsWith('16')) {
```

Comment:
node@17 is out now too so we should figure out how to do >16 instead of `startsWith('16')` here.